testsuite/cluster-test/src/main.rs

```diff
@@ -155,7 +155,11 @@ pub fn main() {
     let mut perf_msg = None;
 
     if args.health_check {
-        runner.run_health_check();
+        let duration = Duration::from_secs(args.duration);
```

Comment:
@andll @ankushagarwal is accuracy of duration unit by secs enough?

Reply:
Yes